.github/workflows/gif_check.yml

```
@@ -0,0 +1,22 @@
+name: "No GIFs"
+
+on: [pull_request]
+
+jobs:
+  file_existence:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Check file existence
+        id: check_files
+        uses: andstor/file-existence-action@v1
+        with:
+          files: "*.gif"
```

```suggestion
          files: "**/*.gif"
```
Let's see if a broader glob pattern helps